src/digraphs.c

```diff
@@ -896,24 +905,24 @@ static Obj FuncDIGRAPH_SYMMETRIC_SPANNING_FOREST(Obj self, Obj adj) {
       stack[1] = 1;
 
       while (1) {
-        j = stack[0];
-        k = stack[1];
+        j   = stack[0];
```

Comment:
I'm using clang-format, which is configures in the file `src/.clang-format`. This means that beyond the choices made in the configuration file (which were meant to be roughly the same choices as we were (mostly) using already), all I am doing is `clang-format -i src/*`, which automatically reformats all the kernel files.